docs/Guide/application-commands/application-command-registry/registering-context-commands.mdx

```diff
@@ -97,68 +87,25 @@ export class SlashCommand extends Command {
   }
 
   public override contextMenuRun(interaction: Command.ContextMenuInteraction) {
-    // Use `isUserContextMenu` to ensure this command was executed as a User Context Menu Command.
-    if (interaction.isUserContextMenu()) {
-    }
+    // ...
   }
 }
 ```
 
-## Context Menu Command Registry Methods
-
-:::warning
+:::caution
 
-To register a context menu input command, you need to acquire the [`ApplicationCommandRegistry`][acr]. Please see the
-[`ApplicationCommandRegistry`][aaacr] section for more details.
+Just like the Message Context Menu Command, the [`registerContextMenuCommand`][rcmc] method takes in a second options
+parameter. Please see [Chat Input Command Registry Options][cicro], since there are important details not repeated here.
 
 :::
 
-### `registerContextMenuCommand`
-
-The method which is responsible for telling the registry that this command should be registered. It takes two
-parameters:
-
-- `command`: The Context Menu Command data to be registered. This can be a `discord.js`
-  [`ApplicationCommandData`][djs-command-data-docs] object a `@discordjs/builders`
-  [`ContextMenuCommandBuilder`][djs-context-menu-command-builder-docs] or a method which returns the
-  `@discordjs/builders `[`ContextMenuCommandBuilder`][djs-context-menu-command-builder-docs].
-
-  For more details on how to use
-  `@discordjs/builders `[`ContextMenuCommandBuilder`][djs-context-menu-command-builder-docs], please see
-  [`discord.js guide`][djs-context-menu-command-builder-docs].
-
-- `options` These are optional registry [`options`][cicro] you can provide when registering the command you provided. By
-  default, this is set to `{ behaviorWhenNotIdentical: RegisterBehavior.Overwrite }`.
```

Comment:
same

Reply:
The arguments to `registerContextMenuCommand` are already talked about in registering Chat Input Commands, as long as we keep the red warning at the top I'd rather not repeat this information.